compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangNodeAnalyzer.java

```diff
@@ -612,4 +613,7 @@
     public abstract void visit(BLangUserDefinedType node, T data);
 
     public abstract void visit(BLangValueType node, T data);
+
+    public abstract void visit(BLangMemberTypeNode node, T props);
```

Comment:
Just to be consitent
```suggestion
    public abstract void visit(BLangMemberTypeNode node, T data);
```